src/patterns/ec2-app/base.test.ts

```diff
@@ -817,48 +817,4 @@ describe("the GuEC2App pattern", function () {
       ],
     });
   });
-
-  describe("GuNodeApp", () => {
```

Comment:
These get moved into their own dedicated test file.

Reply:
👍  - I like the `framework` naming too!

Reply:
> 👍 - I like the `framework` naming too!

Phew! I spent more time worrying about the clarity of this than I should have! Sure `Play` is a framework, but `Node`? That's a runtime, no? Couldn't think of anything better, and we might actually remove the classes later as they don't do much anyway 😂 .